contrib/ubase/patches/getty.patch

```diff
@@ -0,0 +1,103 @@
+Index: ubase/ubase/getty.c
+===================================================================
+--- ubase.orig/ubase/getty.c
++++ ubase/ubase/getty.c
+@@ -10,13 +10,12 @@
+ #include <stdlib.h>
+ #include <string.h>
+ #include <unistd.h>
+-#include <utmp.h>
+ 
+ #include "config.h"
+ #include "util.h"
+ 
+-static char *tty = "/dev/tty1";
+-static char *defaultterm = "linux";
++static char *tty = "/dev/uart";
```

Comment:
It seems to me, that `_PATH_TTY` should be left as it is, because this should be the definition of a path representing controlling terminal for each process (in NetBSD we have the same value: http://bxr.su/NetBSD/include/paths.h#98) and we shouldn't change it to a device file name.

Reply:
Hmm... I'm afraid we don't have any support for `/dev/tty`. Would you be interested in implementing it? My best guess is that it should not be more complex than our `dev_procstat.c`. PTAL at [tty_tty.c](http://bxr.su/FreeBSD/sys/kern/tty_tty.c) from FreeBSD and [tty_tty.c](http://bxr.su/NetBSD/sys/kern/tty_tty.c) from NetBSD.

If you decide to implement it then we should meet to figure out what is the right strategy to follow.

Reply:
I think that actually we do have some support for `/dev/tty`- https://github.com/cahirwpz/mimiker/blob/master/sys/kern/tty.c#L1177. We even use this device file while getting a password from a user - https://github.com/cahirwpz/mimiker/blob/master/lib/libc/gen/getpass.c#L96.

Reply:
Ok. I think I needed that kind of refresher. Seems like my memory is good but short ;-)

Since `tty` variable is a default value for terminal device let's leave it as is.